sdk/tables/azure-data-tables/azure/data/tables/_table_shared_access_signature.py

```diff
@@ -156,12 +156,15 @@ class TableSharedAccessSignature(SharedAccessSignature):
     signature tokens with a common account name and account key.  Users can either
     use the factory or can construct the appropriate service and use the
     generate_*_shared_access_signature method directly.
+
+    :param credential: The credential used for authenticating requests.
+    :type credential: ~azure.core.credentials.AzureNamedKeyCredential
     """
 
     def __init__(self, credential):
         """
-        :param credential: The credential used for authenticating requests
-        :type credential: :class:`~azure.core.credentials.NamedKeyCredential`
+        :param credential: The credential used for authenticating requests.
```

Comment:
Do we need to dup it?

Reply:
From this guidline looks we should: https://review.learn.microsoft.com/en-us/help/onboard/admin/reference/python/documenting-api?branch=main#constructor-parameters
But I'll remove the docstring in constructor as I think it is actually not required in next pylint.